documentation/src/main/asciidoc/topics/ref_logs_xsite.adoc

```diff
@@ -0,0 +1,65 @@
+[id='xsite_logs-{context}']
+= Cross-Site Log Messages
+{brandname} prints different log messages related to cross-site replication.
```

Comment:
@pruivo hey I think there are other trace messages that we might want to add here. For example "IRAC operation failed!" but I couldn't find any identifier (ISPN0000) and I'm not quite sure about them. Do you think we should add those messages here?